starter-code/index.html

```diff
@@ -2,30 +2,94 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel = "stylesheet" href = "css/main.css">
 </head>
 <body>
-  Instagram
 
-  Capture and Share the World's Moments
+  <div class = "overlord">
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+      <div class = "tophalf">
+      
+          
 
-  Oh yeah, did we mention it's free?
+      </div>
+      
 
-  Download on the App Store
-  Get it on Google Play
+    
+      <div class = "bottomhalf">
 
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
 
-  &copy; 2014 Instagram
+
+      </div>
+
+
+
+
+      <div class = "iphone_photo">
+        <img src = "images/phones.png" alt = "iphones">      
+      </div>
+
+
+
+
+      <div class = "instagram_info">
+
+          <div class = "instagram_logo">
+              <img src = "images/brand.png" alt = "instagram logo">
+      
+             <div class = "home_button">
+                <img src = "images/home.png" alt = "home"> 
+                <button class = "btn">Log in</button>
+             </div>
+      
+            </div>
+
+
+         <h1> Capture and Share the World's Moments</h1>
+
+          <p>Instagram is a <b>fast, beautiful</b> and <b>fun</b> way to share your life with friends and family.</p>
+
+          <p>Take a picture or video, choose a filter to transform its look and feel, then post to 
+            Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.</p>
+
+           <p id = "bottomp">Oh yeah, did we mention it's free?</p>
+
+         <img src = "images/badge-iphone.png" alt = "Download on the App Store">
+         <img src = "images/badge-android.png" alt ="Get it on Google Play">
+
+
+      </div>
+
+
+
+
+
+
+
+
+
+
+
+<div class ="clearfix"> </div>
+
+
+  <footer>
```

Comment:
Since we have only one footer by page, you could've manipulated with the tag itself through CSS without introducing _div_ tag and class _floor_.